spec/features/admin/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper"
+
+feature "Managing users" do
+  include AuthenticationWorkflow
+
+  context "as super-admin" do
+    before { quick_login_as_admin }
+
+    describe "creating a user" do
+      it "shows no confirmation message to start with" do
+        visit spree.new_admin_user_path
+        expect(page).to have_no_text "Email confirmation is pending"
+      end
+
+      it "works" do
```

Comment:
Unless there are too many things to mention, it might be nice to have a more descriptive title here?

How about "confirms successful creation with a flash message"?

Reply:
Thank you Rob, I was struggling with naming it. I didn't want to write "creating a user creates a user". But "creating a user confirms successful creation" is better than "creating a user works". I added a commit for that.